internal/database/models/models.go

```diff
@@ -25,6 +25,7 @@ type Story struct {
 	Summary         string  `gorm:"type:text" json:"summary"`
 	Latitude        float64 `gorm:"not null" json:"latitude"`
 	Longitude       float64 `gorm:"not null" json:"longitude"`
+	ImageURL        string  `json:"image_url"`
```

Comment:
make this `type:text` because URLs can be much longer and text should fit it all